### DISCUSSION TITLE
HPC installation in Tianhe2 HPC system

### DISCUSSION THREAD

**Discussion by ryupuriwat** [mdolab/dafoam#615] [Installation]
I am attempting to install DAFoam on the HPC system. The image displays the required modules for loading.
![image](https://github.com/mdolab/dafoam/assets/109540234/f507edf7-b3ff-49bf-8c93-32207a25917c)
I am trying to understand the purpose of each module. It seems that 'intel/18.0.2' can be replaced with 'GCC', and 'impi' can be replaced with 'MPI'. Since Tianhe2 does not have modules such as 'TACC', 'git', 'libfabric', 'boost', 'petsc', or 'python', I need assistance from the professor to determine which module from the available list below should be used for Tianhe2. Additionally, if there is an installation process specific to Tianhe2, could you please provide it?
![image](https://github.com/mdolab/dafoam/assets/109540234/7daeb6e4-c2c5-4a66-b835-90486da50fa5)
![image](https://github.com/mdolab/dafoam/assets/109540234/cb941171-dd97-42d3-95ee-c056eb903438)

**Comment by friedenhe** [comment]
Yeah, the HPC installation guide is for TACC Stampede 2. We don't have guidelines for other HPC systems. You need to adjust it accordingly. There is no need to load modules such as TACC.
